### PR TITLE
Docs: Update Creating-a-scene.html

### PR DESCRIPTION
**Pull request by montoyamoraga**
change "var" to "let"

### CODE REVIEW

**Review by geektcp — COMMENTED** [review]
LGTM

File: docs/manual/en/introduction/Creating-a-scene.html [hunk 1]
```diff
@@ -43,10 +43,10 @@ <h2>Creating the scene</h2>
 		<p>To actually be able to display anything with three.js, we need three things: scene, camera and renderer, so that we can render the scene with camera.</p>
 
 		<code>
-		var scene = new THREE.Scene();
-		var camera = new THREE.PerspectiveCamera( 75, window.innerWidth / window.innerHeight, 0.1, 1000 );
+		let scene = new THREE.Scene();
```
Comment (Mugen87) [inline docs/manual/en/introduction/Creating-a-scene.html:46]: I would say `const` seems more appropriate here. Besides, there are other code sections in this document which should also be updated.

Comment (mrdoob) [reply docs/manual/en/introduction/Creating-a-scene.html:46]: Yep. `const` is the right one here.

Comment (montoyamoraga) [reply docs/manual/en/introduction/Creating-a-scene.html]: thanks! i made a new commit, to replace "let" with "const"

Comment (Mugen87) [reply docs/manual/en/introduction/Creating-a-scene.html]: Please also update the remaining `var` usage in this document.

Comment (montoyamoraga) [reply docs/manual/en/introduction/Creating-a-scene.html]: ok thanks, just did it

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**